cg/cli/workflow/microsalt/base.py

```diff
@@ -140,6 +140,3 @@ def run(context: click.Context, dry_run: bool, config_case_path: click.Path, tic
         str(fastq_path.absolute()),
     ]
     process.run_command(parameters=analyse_command, dry_run=dry_run)
-
-
-microsalt.add_command(store_cmd)
```

Comment:
Do we really want to remove also the store command? I think this one was in use (the others were not)

Reply:
What confused me was that the command where added in different ways. I updated so that `microsalt.add_command` is used for all commands